x/mm2/keeper/keeper.go

```diff
@@ -0,0 +1,181 @@
+package keeper
+
+import (
+	"fmt"
+
+	"cosmossdk.io/collections"
+	"cosmossdk.io/core/store"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/skip-mev/slinky/x/mm2/types"
+)
+
+// Keeper is the module's keeper implementation.
+type Keeper struct {
+	cdc codec.BinaryCodec
+
+	// module authority
+	authority sdk.AccAddress
```

Comment:
How is this different from the authorities defined in params?

Reply:
This is the authority in charge of params (typically gov).  Because of this, the gov params have the final say on who the market authorities are

Reply:
I still think we want 2 authorities in params and 1 here.

Keeper authority: controls who the admin is
Admin auth: controls who the participants are
Participants: Can update the market map